conf/armv7_stretch.src

```diff
@@ -1,5 +1,5 @@
-SOURCE_URL=https://github.com/YunoHost-Apps/synapse_python_build/releases/download/v0.34.0/matrix-synapse_0.34.0-stretch-bin1_armv7l.tar.gz
-SOURCE_SUM=db28673e98572d104cfb58b4a4faa27a2e856fdd3c981f4b5addc26b7be66939
+SOURCE_URL=https://github.com/YunoHost-Apps/synapse_python_build/releases/download/v0.34.1.1/matrix-synapse_0.34.1.1-stretch-bin1_armv7l.tar.gz
```

Comment:
There's no such release in this repository.

Reply:
Oh sorry I didn't realise we also need to prepare a release for that.

@Josue-T I don't know how that works, can you do it ?

Reply:
Yes, I launched the build since you made your issue. 

Just wait until I push everything